crates/subspace-farmer/src/cluster/controller.rs

```diff
@@ -37,6 +38,10 @@ use subspace_rpc_primitives::{
 };
 use tracing::{debug, error, trace, warn};
 
+/// Special "cache group" that all controllers subscribe to and that can be used to query all cache
+/// groups
+const GLOBAL_CACHE_GROUP: &str = "_";
```

Comment:
It looks like each query is directed to a randomly chosen cache group, rather than broadcast to every cache group?

```suggestion
/// Special "cache group" that all controllers subscribe to and that can be used to query any cache
/// group. The cache group for each query is chosen at random.
const GLOBAL_CACHE_GROUP: &str = "_";
```